bundles/AdminBundle/Controller/Admin/DataObject/ClassificationstoreController.php

```diff
@@ -788,13 +788,19 @@ public function relationsActionGet(Request $request)
         $limit = 15;
```

Comment:
not sure if we even this, IMHO this should work without the default. wdyt

Reply:
I have no idea what the background idea of this limit is. I would leave it in for safety, unless you know exactly that it can not cause problems somewhere else ;-)